# SenderId determines the system, the sender of this message is running on. 
# Example: If a process manager is running on the robot nase this is set to 9
agent_id/AgentID sender_id

# ProcessStats is a list of statistical information about a specific process, running 
# under a specific ROBOT-Environment variable / on a specific robot
ProcessStat[] process_stats